src/content/docs/infrastructure/host-integrations/host-integrations-list/ansible-controller-integration.mdx

```diff
@@ -0,0 +1,180 @@
+---
+title: Red Hat Ansible Automation Controller integration
+tags:
+   - Ansible controller  integration
+   - New Relic integrations
+metaDescription: Install our Ansible Automation Controller dashboards and see your Ansible data in New Relic.
+---
+
+import infrastructureAnsibleController from 'images/infrastructure_screenshot-full_ansible-controller.png'
+
+With our Red Hat Ansible Automation Platform dashboards, you can easily track your Ansible Automation Platform logs, keep an eye on your jobs and monitor instance memory.
+
+Built with our infrastructure agent and our nri-prometheus integration, the Ansible Automation Platform integration gives you a set of pre-built dashboards that let you view your most critical query data, all in one place.
+
+<img
+  title="Ansible Controller"
+  alt="A screenshot depicting the Ansible Controller dashboard"
+  src={infrastructureAnsibleController}
```

Comment:
@homelessbirds We did title changes on Ansible autmation controller to Red Hat Ansible automation controller, So please replace the dashboard image from latest screenshot of [Dashboard](https://onenr.io/08wpx4mv6wO).

Reply:
this is done :)